tests/test_globalization.py

```diff
@@ -168,61 +164,60 @@ def run_charset_validation(self, charset):
             shutdown(client)
 
 
-class GlobalizationMetadataTests(GlobalizationTests, unittest.TestCase):
-    @classmethod
-    def setUpClass(cls):
-        cls.test_db = create_test_db()
-
-    @classmethod
-    def tearDownClass(cls):
-        clean_up_test_db(cls.test_db)
+class TestGlobalizationMetadata(GlobalizationTests):
+    @staticmethod
+    @pytest.fixture(scope="class")
+    def test_db():
+        db = create_test_db()
+        yield db
```

Comment:
Is this like a try/finally implementation? If error is thrown / test fails, does the cleanup still happen?

Reply:
Pytest fixtures appear to be an improved version over the classic setup and teardown style methods of unit tests. Fixtures ensure cleanup as well. More info [here](https://docs.pytest.org/en/latest/fixture.html) if you're curious.